mkdocs.yml

```diff
@@ -33,6 +33,7 @@ nav:
   - Examples:
     - EchoServer: examples/echo_server.md
     - gRPCServer: examples/grpc_server.md
+    - RBAC for Secrets: examples/secrets_access.md
```

Comment:
i think `RBAC to access OIDC Secret` or `RBAC to access Secret` might be better

Reply:
done